vscode-test-runner/workspace-1/.joyride/src/integration_test/runner.cljs

```diff
@@ -55,11 +61,18 @@
       (require '[integration-test.require-js-test])
       (require '[integration-test.require-extension-test])
       (require '[integration-test.npm-test])
+      (require '[integration-test.nrepl-start-stop-test])
+      (require '[integration-test.nrepl-eval-test])
       (cljs.test/run-tests 'integration-test.activate-test
                            'integration-test.scripts-test
                            'integration-test.require-js-test
                            'integration-test.require-extension-test
-                           'integration-test.npm-test))
+                           'integration-test.npm-test
+                           ;; TODO: Figure out why this gives:
```

Comment:
Still some TODOs here?

Reply:
Thanks. Forgot.

Reply:
It turns out I did manage to fix the problem with start/stop in my earlier attempts in this PR, I just tested it wrongly. Now added some identical start/stop tests, just to guard against regression in resolving the start/stop promises.

I tried to make an explicit test for the ”server already running” exception, but my `not-thrown?` multi-method caused a strange error, probably because of my lacking macro-fu. Thinking it is OK just to see that we get results, because that means no exception is thrown.